src/rpc/multipass.proto

```diff
@@ -176,9 +176,14 @@ message InfoRequest {
     int32 verbosity_level = 2;
 }
 
+message MapPair {
```

Comment:
To be consistent when you rename `id_map_list` to `id_mappings`, let's call this `IdMap`.

Reply:
Done!